asctec_msgs/GPSData gpsdata
---
sensor_msgs/NavSatFix fix
